source/includes/tutorial1/_training.html.md

```diff
@@ -0,0 +1,69 @@
+# Train Your BRAIN
+
+Now that you’ve written a curriculum of machine teaching (through your Inkling code) to connect your simulation to the Bonsai AI Engine, it’s time to prepare a new version for training. Since we previously created a new BRAIN for training, the `--brain` argument in the below commands is optional, but we’ve left it in to remind you which BRAIN the CLI is targeting.
+
+```shell
+# Push the edited files to the server
+bonsai push
+
+# Start a new BRAIN version
+bonsai train start
+```
+
+Use [`bonsai push`][1] to upload your edited Inkling file whenever you make changes (make sure you filled in the action and state schemas first or you will get an error!) to the server.
+
+> Python 2
+
+```shell
+python move_a_point_sim.py --brain=move-a-point
+```
+> Python 3
+
+```shell
+python3 move_a_point_sim.py --brain=move-a-point
+```
+
+Once you have started training mode with [`bonsai train start`][2] it's time to start running your simulator by calling Python and then the simulator file. Training will begin automatically after you connect your simulator.
```

Comment:
"...it's time to start your simulation."  (You're not "calling Python and then the sim file". )